src/sentry/demo/data_population.py

```diff
@@ -77,6 +77,7 @@
         "jennifer.song",
         "Jen Song",
     ),
+    ("roggenkemper", "Richard Roggenkemper"),
```

Comment:
@roggenkemper do you think you could replace Jen with Zach now that he's our new manager?

Reply:
Yup! you caught that I added myself already lol